Sources/SwiftKotlinFramework/utils/AST+Operations.swift

```diff
@@ -62,6 +62,10 @@ extension StructDeclaration.Member {
         if let variable = declaration as? VariableDeclaration {
             return variable.isStatic
         }
+        if let const = declaration as? ConstantDeclaration {
+            // this is static too:
+            return const.modifiers.isStatic
```

Comment:
Constants in kotlin do not need to be static. Making them static change meaning becuase it allows it to be accessible statically instead of on an instance level right? maybe I am missing something here